package.json

```diff
@@ -82,6 +82,7 @@
     "lodash.merge": "^4.6.2",
     "moment": "^2.29.3",
     "node-libs-react-native": "^1.2.1",
+    "pod-install": "^0.1.38",
```

Comment:
this should not be needed? what platform are you using?

in any case, if needed, it should be a dev-dependancies not a dependancy

Reply:
removed this dependency